.travis.yml

```diff
@@ -15,10 +15,6 @@ cache:
     - $HOME/.composer/cache
     - $HOME/.cache/pip
 
-env:
-  global:
-    - SYMFONY_DEPRECATIONS_HELPER=weak
```

Comment:
This is related to SF 2.8 and 3.0.

But, I think we should keep it and prepare for 3.1